tests/unit_tests/conftest.py

```diff
@@ -82,6 +82,11 @@ def app() -> Iterator[SupersetApp]:
     app.config["PREVENT_UNSAFE_DB_CONNECTIONS"] = False
     app.config["TESTING"] = True
 
+    # loop over extra configs passed in by tests
+    if request and hasattr(request, "param"):
+        for key, val in request.param.items():
+            app.config[key] = val
+
```

Comment:
@betodealmeida I extended this method so that you can pass in extra configs from tests.

Reply:
Oh, I like the idea!

Should we give this a more descriptive name, just in case? Eg, `override_app_config` instead of `param`?

Also, how is this used? Is `request` a Flask request object?

Reply:
I added typing here. Request is a pytest fixture sub request object.